apps/src/code-studio/components/progress/TeacherUnitOverview.tsx

```diff
@@ -221,9 +222,7 @@ const initializeRedux = (
     );
   }
 
-  if (unitData.has_verified_resources) {
-    dispatch(setVerifiedResources(true));
-  }
+  dispatch(setVerifiedResources(!!unitData.has_verified_resources));
 
   if (unitData.is_verified_instructor) {
     dispatch(setVerified());
```

Comment:
The `verifiedInstructor` status shouldn't change mid session, so I don't think we need to worry about this conditional